api/types/src/move_types.rs

```diff
@@ -237,6 +239,81 @@ impl TryFrom<AnnotatedMoveStruct> for MoveStructValue {
     }
 }
 
+impl TryFrom<RawMoveStruct> for MoveStructValue {
+    type Error = anyhow::Error;
+
+    fn try_from(s: RawMoveStruct) -> anyhow::Result<Self> {
+        let mut map = BTreeMap::new();
+        if let Some(tag) = s.variant_info {
+            map.insert(
+                IdentifierWrapper::from_str("__variant_tag__")?,
```

Comment:
Before you use `__variant__`, and here `__variant_tag__`, I think you can just use one, it is the same thing?

Reply:
`__variant__` is a symbolic name, `__variant_tag` a number. Would we ever convert this JSON to other typed PLs, this would be relevant to distinguish.